### ISSUE TITLE
Hover Points/Cells On Function

### ISSUE THREAD

**Issue by dai-zijian** [Kitware/paraviewweb#517]
Hello everyone! 
There is Hover Points/Cells On Function In the Qt version paraview and it's very useful.
![image](https://user-images.githubusercontent.com/56720765/90947389-12b7f980-e468-11ea-970c-1cb8ad159d47.png)
I am developing an application base on paraviewweb and visualizer and I want to import this function. Does anyone know how to do that?

**Comment by jourdain** [comment]
Yes that is possible. You need to implement the selection on the server side and send back to the client the info that you get from that selection along with pushing `picking` action on mouse move.
